CHANGELOG.md

```diff
@@ -4,6 +4,8 @@
 
 ### Added
 
+- aws resources: `aws_autoscaling_schedule`
+  ([PR #194](https://github.com/cycloidio/terracognita/pull/224))
```

Comment:
Move this to the `Unreleased` section :)

Reply:
oups >.<